src/components/form/text-area/index.tsx

```diff
@@ -59,8 +61,10 @@ export const TextArea = React.forwardRef<HTMLDivElement, TextAreaProps>(
     };
 
     React.useEffect(() => {
-      recalculateHeight();
-    }, []);
+      if (value && counter === 0) {
+        recalculateHeight(String(value));
```

Comment:
why do you pass `String(value)` to `recalculateHeight`?

Reply:
because `value` has typ `string | number | readonly string[] | undefined`